modules/gltf/gltf_state.cpp

```diff
@@ -31,6 +31,7 @@
 #include "gltf_state.h"
 
 void GLTFState::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("set_extension_as_used", "extension_name", "required"), &GLTFState::set_extension_as_used);
 	ClassDB::bind_method(D_METHOD("get_json"), &GLTFState::get_json);
```

Comment:
Is there an equivalent of `get_supported_extensions` for gltf state?

Reply:
To import, we need to *support* extensions (a GLTFDocumentExtension can list what it supports with `get_supported_extensions`).
When exporting, we selectively *use* extensions (a GLTFDocumentExtension can mark an extension as used with `set_extension_as_used`).
The list of used extensions is store in GLTFState, but support only exists inside of GLTFDocumentExtension.